plugins/modules/yum_versionlock.py

```diff
@@ -25,7 +25,7 @@
 options:
   name:
     description:
-      - Package name or a list of package names with optional wildcards.
+      - Package name or a list of package names with optional version or wildcards.
```

Comment:
Let's be explicit about since when this is supported:
```suggestion
      - Package name or a list of package names with optional version or wildcards.
      - Specifying versions is supported since community.general 7.2.0.
```

Reply:
ok, added.